### PR TITLE
Rebalance makeshift tools, remove makeshift hammer

### PR DESCRIPTION
**Pull request by Thesingleworstcoder**
Second time's the charm!

## About The Pull Request

For a long time makeshift tools have been in a sorry state. They are (currently) completely useless to everyone except prisoners, and even then most antags in jail have better ways to escape, and actual prisoners aren't allowed to use the tools to escape.
This PR is my attempt to rebalance makeshift tools into something that may actually see use in-game, improving their stats and making them easier to craft, as well as streamlining the process by removing the stupid, awful, bad, useless hammer from the face of the earth. The tools now provide a way to bypass the autolathe, providing for new and exciting gameplay chances!

## How This Contributes To The Skyrat Roleplay Experience

It somewhat assists the problem of people being forced to LRP and steal, borrow and beg for tools from one of the main departments. With that in mind, I expect this to be of most use to people in the service department, as they often struggle to find tools outside of the very limited tool storage and the dark, damp maint tunnels.

## Changelog

:cl:
del: Removed the makeshift hammer
balance: Increased makeshift welders fuel from 5 to 10, putting it imbetween the emergency welder and the actual welder
balance: Reduced the size of the crowbar from bulky to normal, allowing it to be placed into toolbelts
:cl:

### CODE REVIEW

**Comment by Thesingleworstcoder** [comment]
_Okay, let's try this one more time!_
(_Sorry for requesting this again, the last time I tried it went error-y and I panicked and closed the PR. Code's fixed now_)

**Review by GoldenAlpharex — APPROVED** [review]
Looks alright. Better than last time, because that silly hammer is gone.

**Comment by Grunnyyy** [comment]
As it was said in the previous PR by someone else, their main use is Prison or clutch usage, as such they are meant to be a lot slower and more conspicuous than normal tools. Obtaining the Materials required for the tools is no easy task aswell and if you outside of Prison, have ran out of tools (There are several public tools and there is also still the Tool Vendor and the odd maintenance toolbox and locker) then likely the extra materials flying around to craft the makeshift tools are also taken already. The existence alone is already an improvement of a status Quo of Vanilla TG (No Makeshift tools at all)

The Fuel increase I am also against, due to the fact that an Emergency Welder at least uses welding fuel, the Makeshift Welder in fact uses an Emergency Oxygen Tank for fuel wich is.... very far from welding fuel.

As for the Hammer, I personally don't care about it too much, its mostly flavour and adds some kinda realism after it (A crowbars application in the creation of the tools is questionable), but I'd recommend that any Crowbar can be used for crafting by making it a generic Crowbar craft, since someone might already own the pretty common findable crowbar and you have to make a makeshift Crowbar, just to make other makeshift tools.




However to the Toolfix, I would honestly recommend mapping in a Blue Mechanic Toolbox into every Department, for basic Departmental tools.

**Comment by Thesingleworstcoder** [comment]
> 
> 
> As it was said in the previous PR by someone else, their main use is Prison or clutch usage, as such they are meant to be a lot slower and more conspicuous than normal tools. Obtaining the Materials required for the tools is no easy task aswell and if you outside of Prison, have ran out of tools (There are several public tools and there is also still the Tool Vendor and the odd maintenance toolbox and locker) then likely the extra materials flying around to craft the makeshift tools are also taken already. The existence alone is already an improvement of a status Quo of Vanilla TG (No Makeshift tools at all)
> 
> The Fuel increase I am also against, due to the fact that an Emergency Welder at least uses welding fuel, the Makeshift Welder in fact uses an Emergency Oxygen Tank for fuel wich is.... very far from welding fuel.
> 
> As for the Hammer, I personally don't care about it too much, its mostly flavour and adds some kinda realism after it (A crowbars application in the creation of the tools is questionable), but I'd recommend that any Crowbar can be used for crafting by making it a generic Crowbar craft, since someone might already own the pretty common findable crowbar and you have to make a makeshift Crowbar, just to make other makeshift tools.
> 
> However to the Toolfix, I would honestly recommend mapping in a Blue Mechanic Toolbox into every Department, for basic Departmental tools.

However you never see them being used in the role you've mentioned: Prisoners are discouraged from escaping prison OOC, and antags almost never make it to prison, as they either escape or get killed without enough time left in the round to escape once revived. In their current state these tools are worse than useless, with the spritework leaving something to be desired and not many people even knowing that they exist. They're merely wasted lines of code, bytes that could be repurposed. I say that we either remove them from the game like the waste of space they are, or we improve them slightly to not be on-par with normal tools, but not be useless like they currently are, and that's what I've done. The makeshift tools are about 50% slower than normal tools still. 

Also, in regards to them being conspicuous, if someone's examining a prisoner to see if they've got a crowbar on their belt, it's not much of a stretch for a bag search to be done as well. Guards should be expected to check in-detail for contraband, as it will create better, more believable stories when conflict is created.

**I answered the welder complaint in the old PR as well**

**Comment by Grunnyyy** [comment]
>
> However you never see them being used in the role you've mentioned: Prisoners are discouraged from escaping prison OOC, and antags almost never make it to prison, as they either escape or get killed without enough time left in the round to escape once revived. In their current state these tools are worse than useless, with the spritework leaving something to be desired and not many people even knowing that they exist. They're merely wasted lines of code, bytes that could be repurposed. I say that we either remove them from the game like the waste of space they are, or we improve them slightly to not be on-par with normal tools, but not be useless like they currently are, and that's what I've done. The makeshift tools are about 50% slower than normal tools still.
> 
> Also, in regards to them being conspicuous, if someone's examining a prisoner to see if they've got a crowbar on their belt, it's not much of a stretch for a bag search to be done as well. Guards should be expected to check in-detail for contraband, as it will create better, more believable stories when conflict is created.
> 
> **I answered the welder complaint in the old PR as well**

Don't really care about them being conspicuous or not, its fine either way. 
The main part is them being purposefully shit and their existance is to allow a prison escape and not being in gbj. They are nieche on purpose and them being unused by most prisoners (they have been used before) is a policy issue, not a code one.
Getting tools isn't too much of a hassle and a little talking usually always get you some. In the worst case, tell Cargo you want the tools to do bounties and smack em a Quickie bounty cube or god forbid, pay them a little. I see them being underused, sure, but I also don't feel that they would see a lot more use with these changes. (Pretty sure they can't be refueled using Welding fuel either, wich makes the Emergency welder still superior)

**Comment by Thesingleworstcoder** [comment]
> 
> 
> > However you never see them being used in the role you've mentioned: Prisoners are discouraged from escaping prison OOC, and antags almost never make it to prison, as they either escape or get killed without enough time left in the round to escape once revived. In their current state these tools are worse than useless, with the spritework leaving something to be desired and not many people even knowing that they exist. They're merely wasted lines of code, bytes that could be repurposed. I say that we either remove them from the game like the waste of space they are, or we improve them slightly to not be on-par with normal tools, but not be useless like they currently are, and that's what I've done. The makeshift tools are about 50% slower than normal tools still.
> > Also, in regards to them being conspicuous, if someone's examining a prisoner to see if they've got a crowbar on their belt, it's not much of a stretch for a bag search to be done as well. Guards should be expected to check in-detail for contraband, as it will create better, more believable stories when conflict is created.
> > **I answered the welder complaint in the old PR as well**
> 
> Don't really care about them being conspicuous or not, its fine either way. The main part is them being purposefully shit and their existance is to allow a prison escape and not being in gbj. They are nieche on purpose and them being unused by most prisoners (they have been used before) is a policy issue, not a code one. Getting tools isn't too much of a hassle and a little talking usually always get you some. In the worst case, tell Cargo you want the tools to do bounties and smack em a Quickie bounty cube or god forbid, pay them a little. I see them being underused, sure, but I also don't feel that they would see a lot more use with these changes. (Pretty sure they can't be refueled using Welding fuel either, wich makes the Emergency welder still superior)

There's nothing in the code to say they can't be refuelled, and I don't see why we should want these tools to be useless. Prisoners are left alone so often the quality of the tools really doesn't matter, they'll not be stopped and they'll break out. With that in mind, there's no reason to doom the makeshift tools just because we _think_ that billy no-mates is going to do an escape montage and dissapear. They've got potential to be a cool option for when all else is lost. Maybe power's out and the air's draining, and you just need to craft a crowbar with the scant resources you've got? Maybe you need a tool that can't be traced to any department's lathe to keep Sec guessing? The very idea of a makeshift tool is a fun, roleplay-encouraging one that lends itself to a style of play that we should be encouraging for Skyrat. I see no reason why we can't improve these unloved tools.

File: modular_skyrat/modules/modular_items/code/datums/components/crafting/recipes/recipes_misc.dm [hunk 1]
```diff
@@ -1,25 +1,15 @@
...
-	reqs = list(/obj/item/stack/cable_coil = 1,
-				/obj/item/stack/rods = 2,
-				/obj/item/kitchen/rollingpin = 2)
-	time = 80
-	category = CAT_MISC
-
 /datum/crafting_recipe/makeshift/screwdriver
 	name = "Makeshift Screwdriver"
-	tool_paths = list(/obj/item/hammer/makeshift)
+	tool_paths = list(/obj/item/crowbar/makeshift)
```
Comment (JohnFulpWillard) [inline modular_skyrat/modules/modular_items/code/datums/components/crafting/recipes/recipes_misc.dm:12]: Why isn't this just crowbar? I don't see why you would need already existing makeshift tools to make other makeshift tools, why can't I use a regular crowbar to make a makeshift screwdriver?

Comment (Thesingleworstcoder) [reply modular_skyrat/modules/modular_items/code/datums/components/crafting/recipes/recipes_misc.dm:12]: In-game it's explained that the makeshift crowbar's got a hammerhead on one end, allowing it to be used in-place of the makeshift hammer

**Comment by YakumoChen** [comment]
> people being forced to LRP and steal, borrow and beg for tools

This is actually a contradiction? Stealing, borrowing both lead to more RP. Hell, begging for tools is, by definition, RP. **Petty crime is NOT Low RP.**

By your standard, sitting in an empty room using the crafting menu is _more LRP_ than the issues this PR is supposed to "solve"

Nonetheless there are still alternatives to cargo with two other main departments who can print and provide tools to people who ask, as well as a vendor, I don't see the crew readily ever needing to make or use the makeshift variety if they're not in prison.

**Comment by Thesingleworstcoder** [comment]
> > people being forced to LRP and steal, borrow and beg for tools
> 
> This is actually a contradiction? Stealing, borrowing both lead to more RP. Hell, begging for tools is, by definition, RP. **Petty crime is NOT Low RP.**
> 
> By your standard, sitting in an empty room using the crafting menu is _more LRP_ than the issues this PR is supposed to "solve"
> 
> Nonetheless there are still alternatives to cargo with two other main departments who can print and provide tools to people who ask, as well as a vendor, I don't see the crew readily ever needing to make or use the makeshift variety if they're not in prison.

Fair point, allow me to rephrase. 
People are sometimes asshats about giving tools to other players. If everyone on the station is a sane, rational person then this should be a non-issue, but sadly, that's not always the case. From experience I can think of at least three different times this PR would have saved my life. 

Also, I don't see why people are opposed to this idea. I'm not making it so that prisoners can't use this. I'm only slightly improving them, not making them equal to normal tools.

**Comment by YakumoChen** [comment]
> Also, I don't see why people are opposed to this idea. I'm not making it so that prisoners can't use this. I'm only slightly improving them, not making them equal to normal tools.

The point is that they're obnoxious to make and incredibly shitty. They're designed for prisoners. The whole hammer tool is intentionally there to make it harder to make the other tools. 

Your aim seems to be to solve in game, roleplay issues with code, using reasoning that non-prisoner crew will have a reason to make them, which is not what the feature is designed for.

**Pull request merged**